tests/util/blockdev_utils.rs

```diff
@@ -0,0 +1,74 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+use libstratis::consts::SECTOR_SIZE;
+use libstratis::engine::strat_engine::engine::DevOwnership;
+use libstratis::engine::strat_engine::metadata::SigBlock;
+use std::fs::OpenOptions;
+use std::io::{Read, Seek, SeekFrom};
+use std::io::Write;
+use std::path::Path;
+use std::path::PathBuf;
+use util::test_result::TestError;
+use util::test_result::TestErrorEnum;
+
+use util::test_result::TestResult;
+
+macro_rules! try_clean_devs {
+	($val:expr) => {
+		assert!(match clean_blockdevs_headers($val) {
+		        Ok(_) => true,
+		        Err(e) => {
+		        	error!("Failed to clean header. {:?}", e);
+		        	false
+			    }
+			}
+		)
+    }
+}
+
+#[allow(dead_code)] // Needed to avoid warning on test build
+pub fn get_ownership(path: &PathBuf) -> TestResult<DevOwnership> {
```

Comment:
I think the warning is caused because this method is only used in a test that is currently ignored. The comment could say that.